integrations/client/test_delphi_epidata.py

```diff
@@ -284,21 +284,31 @@ def test_covidcast(self):
       # check result
       self.assertEqual(response_1, {'message': 'no results', 'result': -2})
 
-  @patch('requests.post')
-  @patch('requests.get')
-  def test_request_method(self, get, post):
+  @patch('requests_cache.CachedSession')
+  @patch('requests.Session')
+  def test_request_method(self, _Session, _CachedSession):
     """Test that a GET request is default and POST is used if a 414 is returned."""
-    with self.subTest(name='get request'):
+    with self.subTest(name='get request, no cache'):
+      Session = MagicMock()
+      _Session.return_value = Session
       Epidata.covidcast('src', 'sig', 'day', 'county', 20200414, '01234')
-      get.assert_called_once()
-      post.assert_not_called()
+      assert call('get') in Session.request.call_args_list()
```

Comment:
nit: check that the length of the call args (number of calls) is correct. otherwise lgtm